### DISCUSSION TITLE
Problem launchin containerized zk-sync node with docker on linux.

### DISCUSSION THREAD

**Discussion by DevinBeu** [Cyfrin/foundry-full-course-cu#3031] [General]
When i run `npx zksync-cli dev start` i get the following output in the terminal. 
![dockerproblemzksync](https://github.com/user-attachments/assets/0752a3e9-703a-4f2d-b487-cac11eb9134a)

is sais 'docker-credential-desktop: executable fule not found in $PATH, out:' 
i am on linux mint and installed the docker engine and am not using docker desktop on mac (as is demonstrated in the video). Is this cli tool configured specifically for mac users? Any help would be greatly appreciated!

**Comment by EngrPips** [comment]
Hello @DevinBeu, Patrick asked us not to bother ourselves with this and move on if we are unable to setup a local zksync node.
